exporter/ingest/service.py

```diff
@@ -34,12 +34,14 @@ def complete_job(self, job_id: str):
         job_url = self.get_job_url(job_id)
         self.ingest_client.patch(job_url, json={"status": ExportJobState.EXPORTED.value})
 
-    def get_job_state(self, job_id: str) -> ExportJobState:
-        return self.get_job(job_id).export_state
-
     def get_job(self, job_id: str) -> ExportJob:
         job_url = self.get_job_url(job_id)
-        return ExportJob.from_dict(self.ingest_client.get(job_url).json())
+        return ExportJob(self.ingest_client.get(job_url).json())
+
+    def job_exists(self, job_id: str) -> bool:
+        job_url = self.get_job_url(job_id)
+        response = self.ingest_client.session.get(job_url, headers=self.ingest_client.get_headers())
+        return response.ok
```

Comment:
Would we need the response body? For example in case of an error

Reply:
I don't think we need that for a item exists check.
### response.ok
Returns True if status_code is less than 400, False if not.

- This attribute checks if the status code of the response is between 400 and 600 to see if there was a client error or a server error.
- If the status code is between 200 and 400, this will return True.
- This is not a check to see if the response code is 200 OK.